### ISSUE TITLE
[Feature] Updatenotification

### ISSUE THREAD

**Issue by jwillmer** [hadouken/hadouken#133]
Query the last release on github and compare with current running version.

**Comment by jkaberg** [comment]
+1 or even auto update :smile:

**Comment by vktr** [comment]
Yes, this is a good idea :smile: Auto update should be configurable (and disabled by default) since people might depend on Hadouken running stable in its environment.

**Comment by jkaberg** [comment]
Sounds great!
